packages/e2e-test-app/test/__snapshots__/ButtonComponentTest.test.ts.snap

```diff
@@ -621,7 +621,7 @@ Object {
   "Clip": null,
   "CornerRadius": "0,0,0,0",
   "FlowDirection": "LeftToRight",
-  "Foreground": "#E4000000",
```

Comment:
>E4000000 [](http://example.com/codeflow?start=18&length=8)

Why the color is different? Does it depend on the color scheme of your computer?

Reply:
I'm not entirely sure, something in the combination of updating RNX, VS, and maybe moving from Server 2019 to 2022.